src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4045,12 +4024,42 @@ address MacroAssembler::trampoline_call(Address entry) {
   return call_pc;
 }
 
+address MacroAssembler::load_call(Address entry) {
```

Comment:
I am not sure. How about rename this as something like `indirect_call`?

BTW: I may need to take another look at the update of this PR. Hopefully, I can finish week. Thank you for your patience.

Reply:
The reason I don't like "indirect_call", is that all JALR jumps are indirect:
`The indirect jump instruction JALR (jump and link register)`

Maybe the middle ground: "indirect_load_call()" ?

I can live with indirect_call also, as it's indirect from programmers POV vs movptr + jalr.

Reply:
Or `load_and_call`?

Reply:
Ok, done!